doc/en/user/source/production/troubleshooting.rst

```diff
@@ -47,198 +50,199 @@ All tools work against a live Java Virtual Machine, the one running GeoServer in
 
 jps
 ````
-`jps <http://java.sun.com/javase/6/docs/technotes/tools/share/jps.html>`_ is a tool listing all the Java processing running. It can be used to retried the ``pid`` (process id) of the virtual machine that is running GeoServer. For example::
 
-	> jps -mlv
-	
-	16235 org.apache.catalina.startup.Bootstrap start -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djava.util.logging.config.file=/home/aaime/devel/webcontainers/apache-tomcat-6.0.18/conf/logging.properties -Djava.endorsed.dirs=/home/aaime/devel/webcontainers/apache-tomcat-6.0.18/endorsed -Dcatalina.base=/home/aaime/devel/webcontainers/apache-tomcat-6.0.18 -Dcatalina.home=/home/aaime/devel/webcontainers/apache-tomcat-6.0.18 -Djava.io.tmpdir=/home/aaime/devel/webcontainers/apache-tomcat-6.0.18/temp
-	11521  -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -Djava.library.path=/usr/lib/jni -Dosgi.requiredJavaVersion=1.5 -XX:MaxPermSize=256m -Xms64m -Xmx1024m -XX:CMSClassUnloadingEnabled -XX:CMSPermGenSweepingEnabled -XX:+UseParNewGC
-	16287 sun.tools.jps.Jps -mlv -Dapplication.home=/usr/lib/jvm/java-6-sun-1.6.0.16 -Xms8m
+`jps <http://java.sun.com/javase/6/docs/technotes/tools/share/jps.html>`__ is a tool listing all the Java processing running. It can be used to retried the ``pid`` (process id) of the virtual machine that is running GeoServer. For example::
```

Comment:
```suggestion
`jps <http://java.sun.com/javase/6/docs/technotes/tools/share/jps.html>`__ is a tool listing all the Java processing running. It can be used to retrieve the ``pid`` (process id) of the virtual machine that is running GeoServer. For example::
```